packages/next/src/views/Logout/LogoutClient.tsx

```diff
@@ -18,7 +18,8 @@ export const LogoutClient: React.FC<{
 }> = (props) => {
   const { adminRoute, inactivity, redirect } = props
 
-  const [isLoggedOut, setIsLoggedOut] = React.useState<boolean | undefined>(undefined)
+  const { logOut, user } = useAuth()
+  const [isLoggedOut, setIsLoggedOut] = React.useState<boolean | undefined>(!user)
```

Comment:
This is just to prevent logout requests from happening with errors when the user has already been logged out.